tictactoe/src/main/java/models/GameBoard.java

```diff
@@ -0,0 +1,52 @@
+package main.java.models;
+
+import java.util.List;
+
+public class GameBoard {
+  private int noOfRowsGameBoard;
```

Comment:
The suffix does not add any value here as this is the GameBoard class.

Reply:
If the number of rows and cols are going to be the same then why take 2 params and increase risk of getting different values. Always consider that your class will be instantiated by someone else who does not know the internal implementation.